app/src/main/res/values/styles.xml

```diff
@@ -261,10 +261,10 @@
         <item name="menuItemButtonTintColor">@color/accent_high_contrast_private_theme</item>
 
         <!-- Shared widget colors -->
-        <item name="mozac_primary_text_color">@color/fx_mobile_private_text_color_primary</item>
+        <item name="mozac_primary_text_color">@color/primary_text_normal_theme</item>
```

Comment:
@gabrielluong I had to revert your change due to widget color dependency in A-C.  Not sure if this will break what you wanted to fix.

Reply:
This isn't a complete revert. This used to be `@color/primary_text_private_theme`.